ruff.toml

```diff
@@ -15,48 +15,36 @@ select = [
 explicit-preview-rules = false
 ignore = [
     # Pylint convention
-    "PLC0105", # type-name-incorrect-variance
-    "PLC0415", # import-outside-top-level
-    "PLC2801", # unnecessary-dunder-call
-    "PLC0206", # dict-index-missing-items
-    "PLC1901", # compare-to-empty-string
-    "PLC2701", # import-private-name
-    "PLC0414", # useless-import-alias
+    "PLC0105", # type-name-incorrect-variance (2) (?)
+    "PLC0415", # import-outside-top-level (286) (x)
+    "PLC2801", # unnecessary-dunder-call (11) (x)
```

Comment:
The reason this is a "good" ignore for me is because we often use `object.__setattr__` to edit frozen dataclasses and while streamable exists, I'm not sure that's going away.